### DISCUSSION TITLE
Footnote added to generated documents to show where the source is located

### DISCUSSION THREAD

**Discussion by lmorg** [lmorg/murex#693] [Show and tell]
@orefalo 

A frequent pain you've ran into when contributing to the docs is not knowing where the source files are. So I've added a footer to each document:

<img width="772" alt="image" src="https://github.com/lmorg/murex/assets/12153701/b914f948-a842-44bc-8114-4aef2e4c71c2">

Hope this helps

**Comment by orefalo** [comment]
this helps a LOT
